zh, en: optimize access-dashboard.md (#1507)
pingcap/docs-tidb-operator: en/access-dashboard.md

# Access TiDB Dashboard

TiDB Dashboard is a visualized tool introduced since TiDB v4.0 and is used to monitor and diagnose TiDB clusters. For details, see [TiDB Dashboard](https://docs.pingcap.com/tidb/stable/dashboard-intro).

This document describes how to access TiDB Dashboard in Kubernetes.

- In a test environment, you can [access TiDB Dashboard by port forward](#method-1-access-tidb-dashboard-by-port-forward).
- In a production environment, it is recommended to [access TiDB Dashboard by Ingress](#method-2-access-tidb-dashboard-by-ingress). You can also enable the TLS transfer. See [Enable Ingress TLS](#enable-ingress-tls) for details.
- To access TiDB Dashboard without a domain name, you can [use NodePort Service](#method-3-use-nodeport-service).

> **Note:**
>
> Due to the special environment of Kubernetes, some features of TiDB Dashboard are not supported in TiDB Operator. See [Unsupported TiDB Dashboard features](#unsupported-tidb-dashboard-features) for details.

> In this document, TiDB Dashboard is accessed using the Discovery service. TiDB Operator starts a Discovery service for each TiDB cluster. The Discovery service can return the corresponding startup parameters for each PD Pod to support the startup of the PD cluster. The Discovery service can also send proxy requests to the TiDB Dashboard.

Reviewer: ```suggestion
In this document, you can access TiDB Dashboard using the Discovery service. TiDB Operator starts a Discovery service for each TiDB cluster. The Discovery service can return the corresponding startup parameters for each PD Pod to support the startup of the PD cluster. The Discovery service can also send proxy requests to the TiDB Dashboard.
```

    enableDashboardInternalProxy: true
```

## Method 1. Access TiDB Dashboard by port forward

> **Warning:**
>
> This guide shows how to quickly access TiDB Dashboard. Do **NOT** use this method in the production environment. For production environments, refer to [Access TiDB Dashboard by Ingress](#method-2-access-tidb-dashboard-by-ingress).

TiDB Dashboard is built in the PD component in TiDB 4.0 and later versions. You can refer to the following example to quickly deploy a TiDB cluster in Kubernetes.

> 1. Deploy the following example `.yaml` file into the Kubernetes cluster by running the `kubectl apply -f` command:

Reviewer: 请和开发确认下是否要把 `kubectl apply -f` 改成 `kubectl create -f`
本文有多处用到此命令，如需修改，请全文查找替换。

Author: 见 https://github.com/pingcap/docs-tidb-operator/pull/1507#discussion_r774398545，这里应该是可替换可不替换。

    kubectl port-forward svc/basic-discovery -n ${namespace} 10262:10262
    ```

>     By default, `port-forward` binds to the IP address 127.0.0.1. If you need to use another IP address to access the machine running the `port-forward` command, you can add the `-address` option and specify the IP address to be bound.

Reviewer: ```suggestion
    By default, `port-forward` binds to the IP address 127.0.0.1. If you need to use another IP address to access the machine running the `port-forward` command, you can add the `--address` option and specify the IP address to be bound.
```

3. Visit <http://localhost:10262/dashboard> in your browser to access TiDB Dashboard.

## Method 2. Access TiDB Dashboard by Ingress

> In important production environments, it is recommended to expose the TiDB Dashboard service using Ingress.

Reviewer: ```suggestion
In important production environments, it is recommended to expose the TiDB Dashboard service using `Ingress`.
```

Author: 上下文都没加反引号
